src/utils/utils.ts

```diff
@@ -551,7 +551,15 @@ export function getSourceFromLocations(
 export function runStarkNetClassHash(filePath: string): string {
   const warpVenvPrefix = `PATH=${path.resolve(__dirname, '..', '..', 'warp_venv', 'bin')}:$PATH`;
 
-  const classHash = execSync(`${warpVenvPrefix} starknet-class-hash ${filePath}`).toString().trim();
+  let classHash: string;
+  try {
+    classHash = execSync(`${warpVenvPrefix} starknet-class-hash ${filePath}`).toString().trim();
```

Comment:
you can return from inside the return from inside the try block no need to make let declaration outside

Reply:
Wouldn't be better that exist a difference between the 2 errors thrown in that function? Otherwise the error from the conditional will be caught by the `catch`

Reply:
The error message from the condition on line 555 is not helpful.
We should extract the reason from the error message, this should be done in the catch.

Also there are two cases of the execSync failing, one where the command fails because the starknet-class-hash wasn't found in that we should recommend the user to use `warp install` the other case is where the `starknet-class-hash` command returns an error. This should be made visible to the user rather than just leaving it as `starknet-class-hash` failed